src/devices/bus/nes_ctrl/zapper.cpp

```diff
@@ -73,14 +114,20 @@ void nes_zapper_device::device_reset()
 {
 }
 
+void nes_bandaihs_device::device_reset()
+{
+	m_latch = 0xff;
+	m_strobe = 0xff;
+}
```

Comment:
Same for this one – call the base implementation even if it does nothing right now.

Reply:
Done.